### PR TITLE
Add members wizard integration

### PR DESCRIPTION
**Pull request by buoyad**
- Start setting up store for wizard
- Add stoybook for confirm screen

cc @keybase/y2ksquad 

image

image

### CODE REVIEW

File: shared/teams/add-members-wizard/confirm.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  const onSelectRole = newRole => setRole(newRole)
+  const onConfirmRole = newRole => {
+    setRole(newRole)
+    setShowingMenu(false)
+    dispatch(TeamsGen.createSetAddMembersWizardRole({role: newRole}))
+  }
+  return (
+    <Kb.Box2 direction="horizontal" gap="tiny" alignItems="center">
+      <Kb.Text type="BodySmall">Invite as: </Kb.Text>
+      <FloatingRolePicker
```
Comment (jakob223) [inline shared/teams/add-members-wizard/confirm.tsx]: There's a prop on this to change the text from e.g. 'Make owner' to "Add as owners" I think.

File: shared/teams/add-members-wizard/confirm.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        open={showingMenu}
```
Comment (jakob223) [inline shared/teams/add-members-wizard/confirm.tsx]: Is there another ticket for this behavior?

image
image

Comment (buoyad) [reply shared/teams/add-members-wizard/confirm.tsx]: yep, 1560

File: shared/teams/add-members-wizard/confirm.tsx (hunk 1 above)
Comment (buoyad) [reply shared/teams/add-members-wizard/confirm.tsx]: Fixed in #22980

**Pull request merged**